packages/auto-approve/src/auto-approve.ts

```diff
@@ -162,7 +163,7 @@ export function handler(app: Probot) {
       const prConfig = await getBlobFromPRFiles(
         context.octokit,
         owner,
-        repo,
+        repoBase,
```

Comment:
I *think* we want to keep this one as the head repo - we are trying to get the file from the head (repo that has the changes for the pull request) repo/sha.

Reply:
Oh right. I can't readily tell from the documentation, but it's also not erroring, so I'll leave it as is.